react-client/src/components/Podcasts/Search/Search.js

```diff
@@ -0,0 +1,35 @@
+import React, {useEffect, useState} from "react";
+import style from "./Search.module.css"
+import {searchPodcasts} from "../../../services/Podcasts";
+
+const Search = (props) => {
+    const [searchString, setSearchString] = useState('');
+
+    useEffect(async () => {
+        async function fetchData() {
+            return await searchPodcasts(searchString)
+        }
+
+        try {
+            props.updateSearch(await fetchData())
+
+        } catch (err) {
+            console.error(`${err}`)
```

Comment:
I think it will be better instead of returning 404 from server and getting to the catch block when query is an empty string to return all results in case